nbs/40_tabular.core.ipynb

```diff
@@ -165,7 +165,7 @@
        "      <th>0</th>\n",
```

Comment:
<p>Hey, you could wrap all this into:</p><pre class="ql-syntax" spellcheck="false">with TemporaryDirectory() as d:
  toexport(d/fname)
  new = load(d/fname)
  ...
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2857/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>